heron/tools/tracker/src/python/main.py

```diff
@@ -55,6 +55,8 @@ def __init__(self):
         (r"/topologies/executionstate", handlers.ExecutionStateHandler, {"tracker":self.tracker}),
```

Comment:
IIRC we decided that executionstate would be deprecated in favor of /metadata and /runtimestate. Is that correct? If so would you clarify that in the executionstate class, so it's clear to other readers why we have duplicate info returned by multiple endpoints.